types/identicon.js/index.d.ts

```diff
@@ -0,0 +1,133 @@
+// Type definitions for identicon.js 2.3
+// Project: https://github.com/stewartlord/identicon.js
+// Definitions by: D0miH <https://github.com/D0miH>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Color = [number, number, number, number];
+
+export interface IdenticonOptions {
+    background?: Color;
+    foreground?: Color;
+    margin?: number;
+    size?: number;
+    format?: "svg" | "png";
+}
+
+export class PNGlib {
```

Comment:
PNGlib isn't exported by this package.

Reply:
@rbuckton i am new to typescript so instead of classes in now used interfaces. 
I hope this is the correct to do it.